### ISSUE TITLE
Issues while following documentation of Mnestix complete app example startup

### ISSUE THREAD

**Issue by blaisephilip** [mnestix/mnestix-browser#56]
By attempting to execute the steps [here](https://github.com/mnestix/mnestix-browser/tree/release/mnestix-browser-v1.0.0?tab=readme-ov-file#run-mnestix-as-complete-aas-application), the detected issues:

1. yarn docker:dev will immediately stuck in a **fresh repository clone** due to this:

In package.json the row 81 contains the following:
"docker compose -f docker-compose/compose.dev.yml down mnestix-frontend"
Due to this, the startup by yarn docker:dev is blocked.
(Error: unknown command "mnestix-frontend" for "docker compose down")

Adding case-dependent shutdown of mnestix-frontend container would be better.

2. Active login at docker.io is expected to pull images, not only installation. --> Please add as prerequisite.

3. Yarn install fails due to a dependency within an image build. It would be good to add some user feedback so the currently built services are better indicated in the terminal.

```
 => ERROR [deps 3/3] RUN yarn install --frozen-lockfile --production                                               2.1s
------
 > [deps 3/3] RUN yarn install --frozen-lockfile --production:
#0 0.482 yarn install v1.22.19
#0 0.546 [1/4] Resolving packages...
#0 0.717 warning Resolution field "string-width@4.2.3" is incompatible with requested version "string-width@^5.1.2"
#0 0.720 warning Resolution field "string-width@4.2.3" is incompatible with requested version "string-width@^5.0.1"
#0 0.839 [2/4] Fetching packages...
#0 0.846 warning Pattern ["@mui/styled-engine@latest"] is trying to unpack in the same destination "/usr/local/share/.cache/yarn/v6/npm-@mui-styled-engine-5.15.14-168b154c4327fa4ccc1933a498331d53f61c0de2-integrity/node_modules/@mui/styled-engine" as pattern ["@mui/styled-engine@^5.15.14"]. This could result in non-deterministic behavior, skipping.
#0 0.849 warning Pattern ["wrap-ansi@^7.0.0"] is trying to unpack in the same destination "/usr/local/share/.cache/yarn/v6/npm-wrap-ansi-cjs-7.0.0-67e145cff510a6a6984bdf1152911d69d2eb9e43-integrity/node_modules/wrap-ansi-cjs" as pattern ["wrap-ansi-cjs@npm:wrap-ansi@^7.0.0"]. This could result in non-deterministic behavior, skipping.
#0 1.016 error An unexpected error occurred: "https://registry.yarnpkg.com/@svgr/webpack/-/webpack-8.1.0.tgz: self-signed certificate in certificate chain".
#0 1.016 info If you think this is a bug, please open a bug report with the information provided in "/app/yarn-error.log".
#0 1.016 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
------
failed to solve: process "/bin/sh -c yarn install --frozen-lockfile --production" did not complete successfully: exit code: 1
error Command failed with exit code 17.
```

**Comment by hofermo** [comment]
Hi @blaisephilip,

thank you for your issue, your feedback is very valuable to us 👍 
If you like, it would be very valuable for us [to schedule a meeting](mailto:mnestix@xitaso.com) to better understand your issues, here are some preliminary comments:

1. It looks like your docker version might be a bit outdated, could you try updating it and see if this error persits? The syntax `docker compose up [OPTIONS] [SERVICE...]` should follow the docker specifications:
https://docs.docker.com/reference/cli/docker/compose/up/ 
We will have a look if we should remove the down command at the end.

2. Into what problems did you run, where you had to login at docker.io?
All the images we use are readily available in the docker hub (see [here](https://hub.docker.com/r/mnestix/mnestix-api) and [here](https://hub.docker.com/r/mnestix/mnestix-browser)) and using the `yarn docker:dev` command are built on the fly.

3. We will look into the yarn error, although we haven't encountered it yet. We will try to add some more feedback when building the docker images!


Thank you for your feedback

**Comment by blaisephilip** [comment]
Hi @hofermo 

1. I've upgraded my Rancher V1.14.1 (Docker version 26.1.0-rd) --> Rancher V1.15.0 (Docker version 27.1.1-rd). Error was the same, so I'll keep the docker down-command removed.
Note: From organizational reasons, I shall use a Rancher Desktop instead of Docker Desktop. (OS: Windows 10 Pro)

2. Without active login at docker.io, I could not pull the images yesterday - process went to timeout. Might be in connection with my Rancher or network. Anyway, easy to bypass.

3. The error is not reproducible today, probably it was some temporary issue.

The services started up properly, UI looks nice. I'll try to connect it to my AAS repository.
Operations developed by IDTA are not always straightforward for AAS object management so I find your team's initiative good to build a more user friendly solution.  👍

**Comment by JonathanXITASO** [comment]
Short update from our side. We have released Mnestix version 1.1.0 !
This version also ships with an updated list of yarn commands like `yarn docker:prune` to clean up docker from all mnestix containers, volumes and networks. The `yarn docker:dev` command now no longer contains `docker compose down`.

If you want to use hot reloading for development, you can also check out `yarn docker:backend` and start the frontend service separately in debug mode (see the [readme](https://github.com/mnestix/mnestix-browser?tab=readme-ov-file#getting-started-with-developing)).

We continuously want to improve the user experience, especially for users who are just starting out.
Any feedback is highly appreciated, so thank you very much for your active assistance.
